lang/strings_english.txt

```diff
@@ -1184,6 +1187,7 @@ $s_resolved = 'Resolved';
 $s_total = 'Total';
 $s_balance = 'Balance';
 $s_most_active = 'Most Active';
+$s_most_votes = 'Hot';
```

Comment:
Do you have a suggestion? It just bothers me to name this "Most votes", because currently no other UI mentions anything about votes. It is "bug count" (occurencies) and "Me too". I thought "votes" term would be delayed until a proper +1/-1 votes system will be added.